Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM java:8-jdk
+MAINTAINER gdubina@dataart.com
+
+RUN apt-get update
+RUN apt-get install net-tools
+
+ADD release/restcomm-media-server.zip /tmp/restcomm-media-server.zip
+
+#ADD https://mobicents.ci.cloudbees.com/view/MediaServer/job/RestComm-MediaServer-4.x/lastSuccessfulBuild/artifact/bootstrap/target/restcomm-media-server.zip /tmp/restcomm-media-server.zip
+
+ADD https://mobicents.ci.cloudbees.com/view/MediaServer/job/RestComm-MediaServer-4.x/lastSuccessfulBuild/artifact/media-version.txt /tmp/media-version.txt
+
+ENV work_dir /opt/restcomm-media-server
+
+RUN unzip /tmp/restcomm-media-server.zip -d /opt && \
+    chmod +x $work_dir/bin/run.sh && \
+    rm /tmp/restcomm-media-server.zip && \
+    mv /tmp/media-version.txt $work_dir
+
+ADD scripts/configure.sh $work_dir
+ADD scripts/run.sh $work_dir
```

Comment:
Are these auto-configures scripts that you created? Similar to the ones we have for RestComm?
If so, they should be included with the binary instead.